articles/healthcare-apis/includes/fhir-bulk-delete-operation.md

```diff
@@ -67,18 +68,35 @@ Sample response of successfully completed delete job.
             "part": [
                 {
                     "name": "Practitioner",
-                    "valueDecimal": 10.0
+                    "valueInteger64": 10.0
                 },
                 {
                     "name": "Specimen",
-                    "valueDecimal": 7.0
+                    "valueInteger64": 7.0
                 },
                 {
                     "name": "Device",
-                    "valueDecimal": 3.0
+                    "valueInteger64": 3.0
                 }
             ]
         }
     ]
 }
 ```
+## Bulk Delete operation error messages
+
+Here are the list of error messages that can occur if bulk delete operation fails
+
+|HTTP Status Code | Details | Recommended action |
+|-----------------|---------|--------------------|
+|202 |Bulk delete job is stuck | Cancel the bulk delete job and try again.|
```

Comment:
202 doesn't inherently mean the job is stuck. It is normal for the job to return this status while it is running. If it is taking longer than expected this could show it is stuck. The way to check is to run a FHIR search with the same parameters as the bulk delete job and _summary=count. If the count of resources is going down the job is working.